### PR TITLE
Feat: Add frontend code to encrypt password profiles

### PR DESCRIPTION
**Pull request by biancarosa**
Follow-up on PR #567, contains only the frontend code from that branch.

Leaving it on draft becasue we want #580 to be merged first, and also because I already see some improvements I'd like to make.

### CODE REVIEW

File: packages/lesspass-pure/src/api/user.js [hunk 1]
```diff
@@ -4,6 +4,12 @@ export default {
   login({ email, password }) {
     return http.post("/api/auth/jwt/create/", { email, password });
   },
+  getLoggedUserInformation() {
+    return http.get('/api/auth/users/me');
```
Comment (guillaumevincent) [inline packages/lesspass-pure/src/api/user.js:8]: missing trailing slash see https://github.com/lesspass/lesspass/blob/master/containers/backend/api/tests/test_user.py#L20

File: packages/lesspass-pure/src/api/user.js [hunk 2]
```diff
@@ -4,6 +4,12 @@ export default {
...
+  },
+  patch({ key }) {
+    return http.patch('/api/auth/users/me', { key });
```
Comment (guillaumevincent) [inline packages/lesspass-pure/src/api/user.js:11]: same here

**Comment by guillaumevincent** [comment]
Hello,

is it possible to rebase on top of master and remove the change not required (end of line, extra space, etc) ?

You have also an issue on a missing trailing slash (see my comments)

We need also to test this, maybe with end to end tests. I can do it like on the backend code. Or I can let you add tests. Let me know

Thank you

**Comment by edouard-lopez** [comment]
@guillaumevincent for review purpose, Github let you  _hide whitespace changes_ (and alike) On this particular case we go from 28 to 14 files to review.
hide whitespace changes

**N.B.:** I didn't review the PR

**Comment by guillaumevincent** [comment]
@edouard-lopez yes, but those should not be present in the initial pull request

**Comment by biancarosa** [comment]
Closing here on behalf off #586

**Pull request closed without merging**